CHANGELOG.md

```diff
@@ -32,6 +35,13 @@
   updates to existing configurations.
 - `PATCH /network-interfaces/{id}` only allowed post-boot. Use `PUT` for
   pre-boot updates to existing configurations.
+- MMDS failure response for retrieval of unsupported JSON value types
+  (other than strings and objects) is changed from 500 InternalServerError
+  to 501 NotImplemented, if the request was made for an IMDS response format.
+- MMDS failure response to `PUT` and `PATCH` requests for MMDS data store,
```

Comment:
This can also be shortened to: "Allowed the MMDS data store to be initialized with all supported JSON types. Retrieval of these values (besides String, Array, Dictionary) is only possible in JSON mode."